spring-boot-autoconfigure/src/main/java/org/axonframework/springboot/autoconfig/AxonServerAutoConfiguration.java

```diff
@@ -127,6 +131,13 @@ public CommandPriorityCalculator commandPriorityCalculator() {
         return CommandPriorityCalculator.defaultCommandPriorityCalculator();
     }
 
+    @Bean
+    @ConditionalOnMissingBean
+    public CommandLoadFactorProvider commandLoadFactorProvider(
+            @Value("${axon.command.load-factor:100}") int loadFactor) {
```

Comment:
`load-factor` is exclusively used by Axon Server, I'd expect that to be reflected in the property name.

Reply:
you're right, I'm going to change the property name

Reply:
Done